eng/restore/repoRestore.props

```diff
@@ -1,5 +1,6 @@
 <Project TreatAsLocalProperty="ExcludeRestorePackageImports">
   <PropertyGroup>
+    <ToolSetCommonDirectory>$(RepoRoot)artifacts\toolset\Common\</ToolSetCommonDirectory>
```

Comment:
NormalizePath?

Reply:
I just moved that property from Directory.Build.props into this file as it is only needed for projects that don't support restore (the remaining runtime.depproj and the pkgprojs). No need to change it.